src/components/InlineAlert/InlineAlert.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Canvas, ArgTypes, Story, Primary, Controls } from '@storybook/blocks';
+import * as InlineAlertStories from './InlineAlert.stories';
+import { InlineAlert } from './InlineAlert';
+
+<Meta of={InlineAlertStories} />
+
+<SectionHeader
+  title="Inline Alert"
+  sections={[
+    { title: 'Overview', href: '#overview' },
+    { title: 'Props', href: '#props' },
+    { title: 'Usage', href: '#usage' },
+  ]}
+/>
+
+## Overview
+
+A message from the system to the user that offers contextual and timely information pertaining to the user's current task.
+
+<Primary />
+
+## Props
+
+<Controls />
+
+## Usage
+
+<Tip>
+  The InlineAlert component uses CSS Container Queries for responsive design. To enable this feature
+  wrap the `InlineAlert` in a parent element and set the `container-type` CSS property on this
+  parent element.
+</Tip>
+
+<UsageGuidelines
+  guidelines={[
+    'To provide contextual information to the user pertaining to their current task.',
+    'In conjunction with form validation - When you need to learn about a feature or information that is relevant to your current activity.',
+    'When you need assistance in fixing errors, often in conjunction with form validation.',
+  ]}
+  policies={[
+    'If you need to deliver a global message from the admin to all users (e.g. maintenance update), use a broadcast message instead.',
+    `If you are immediately confirming a user's action while they remain in the same view, use a toast instead.`,
+    'If you are promoting, soliciting feedback, or informing the user about a feature, use a banner instead.',
+  ]}
+/>
```

Comment:
1. for documentation consistency, I suggest we put a new section here with all the variants, in order to have documentation per case (if needed). Check also other mdx files as guide for the structure of the page.

```
<SubsectionHeader title="Variants" />  /** Use this instead of hashtag-header for style purposes */

### Neutral

<Canvas of={InlineAlertStories.Neutral} />

....
```
2. We can merge `success, warning, error, informational, neutral` into one story named "Types" with documentation and rendering of all types in one story. This is an optional suggestion though.

eg:

```
<SubsectionHeader title="Variants" />  /** Use this instead of hashtag-header for style purposes */

### Types

There are 4 types of inline alert based on.........

<Canvas of={InlineAlertStories.Types} />

....
```

3. Really nice and interesting touch on adding controls table instead of `<ArgTypes of={InlineRadio} />`. This one adds our playground logic and the controls to the mdx level and not on a separate story as we already do in all the other components. This is a new documentation feature though and creates inconsistency. We have to either keep this and create ticket to change the documentation for all the other components in order to have the playground in the overview section of the mdx (as in the inline alert), or keep the current logic of having a separate Playground story and change it later for all components. @panvourtsis thoughts?

Reply:
Adjusted documentation to include variants and `ArgTypes`